change/@ni-nimble-components-a30a4473-2c1e-48e6-9fc4-ae68c2617219.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Does this reduction in scope mean that #1011 will be complete as soon as we remove the roundToInteger format?

Reply:
It won't quite be complete, but it'll be really close. I think the open items still are:
- remove roundToInteger format
- localize the column (there's an open PR for this already: #1499)
- add a few things (like alignment) to the Angular directive to ensure it matches the web component API
- implement Blazor support